ui/react-components/components/Provider/ProviderSearch.spec.jsx

```diff
@@ -44,16 +44,36 @@ describe('Provider Search', () => {
         }];
         const selectedProvider = "Provider One";
         const onChangeSpy = jest.fn();
-        const {container, getByText} = renderWithReactIntl(
+        const {container,getByText} = renderWithReactIntl(
             <ProviderSearch onChange={onChangeSpy} maxAppointmentProvidersAllowed = {5}
                             selectedProviders={selectedProviders} onProviderRemove={jest.fn()}/>);
-        const inputBox = container.querySelector('.react-select__input input');
-        fireEvent.change(inputBox, {target: {value: "One"}});
-        await waitForElement(() => (container.querySelector('.react-select__menu')));
-        const option = getByText(selectedProvider);
-        fireEvent.click(option);
-        expect(getAllProviders).toHaveBeenCalled();
-        expect(onChangeSpy).toHaveBeenCalledTimes(1);
+        // const inputBox = container.querySelector('.react-select__input input');
```

Comment:
Any specific reason why these lines are commented?

Reply:
Actually this test case is failing. that's why commented. Tried to resolve it yesterday. But need to look again

Reply:
Can you please check the test case. it is failing because of time out?

Reply:
resolved the problem with @meenakshi-dhanani & pushed the code. please check.

Reply:
Thanks! I couldn't reach to a fix in little time which I spent on it. Thanks!